examples/api/lib/material/input_decorator/input_decoration.material_state.0.dart

```diff
@@ -0,0 +1,72 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
I believe that you are the 1st developer other than @gspencergoog to provide an example using the new system!

Reply:
Yes, and unfortunately I've changed the format underneath you. Sorry about that.

Here's what should change:

1) Remove the "description" section (it's redundant with the description in the source)
2) Remove any of the section markers (those bits with ▼▼▼ and ▲▲▲), and the comment block with the "Template" in it.
3) Feel free to remove any extraneous comments.

Take a look at some of the other examples at HEAD to see what it should look like. Do leave the Copyright header, since that needs to be there.

Reply:
I will clean up